cypress/e2e/seededFlows/adminFlows/organizations/manageOrganizations.spec.js

```diff
@@ -1,16 +1,23 @@
 import { verifyAndDismissFlashMessage } from '../shared/utilities';
 
 function openOrganizationOptions(callback) {
+  cy.findByRole('button', { name: 'Options' }).should(
+    'have.attr',
+    'aria-haspopup',
+    'true',
+  );
+  cy.findByRole('button', { name: 'Options' }).should(
+    'have.attr',
+    'aria-expanded',
+    'false',
+  );
+  cy.findByRole('button', { name: 'Options' }).click();
+  expect(
+    cy.findByRole('button', { name: 'Options' }).getAttribute('aria-expanded'),
+  ).to.equal('true');
   cy.findByRole('button', { name: 'Options' })
```

Comment:
not sure if this is the best way to refactor out of the chained "then" syntax, if async/race condition issues present, we may need to refactor according to https://docs.cypress.io/guides/core-concepts/retry-ability#Actions-should-be-at-the-end-of-chains-not-the-middle using a mounted element and a stub